front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -33,9 +32,11 @@ const IdeasNewPage = () => {
   } = useProjectBySlug(slug);
   const { data: authUser } = useAuthUser();
   const { data: phases, status: phasesStatus } = usePhases(project?.data.id);
-  const { phase_id } = parse(location.search, {
-    ignoreQueryPrefix: true,
-  }) as { [key: string]: string };
```

Comment:
Turns out this type assertion was wrong after all, when we hit /ideas/new without a phase_id. Tried to improve and replace with our preferred solution (useSearchParams).